docs/guides/securing-apps/mcp-authz-server.adoc

```diff
@@ -0,0 +1,113 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Integrating with MCP (Model Context Protocol)"
+priority=115
+summary="Using {project_name} as an authorization server in MCP">
+
+There are the following four MCP versions:
+
+* 2024-11-05 (initial version)
+* 2025-03-26
+* 2025-06-18
+* 2025-11-25 (latest version)
+
+The initial version is outdated, so this guide treats other three versions. The guide tells you the following:
+
+* Which MCP version {project_name} supports.
+* How to set up {project_name} as an authorization server in MCP.
+
+== Standards Compliance MCP requires
```

Comment:
Please add `OAuth 2.0 Protected Resource Metadata (RFC 9728)` as MUST, and in Keycloak column add `Requires workaround`

Reply:
RFC 9728 is for a resource server, namely MCP server. It is not for an authorization server. Therefore I did not add RFC 9728 here.